src/g1.rs

```diff
@@ -167,6 +170,16 @@ where
     }
 }
 
+#[cfg(feature = "zeroize")]
+// manual implementation because 'subtle' doesn't have a zeroize feature
+impl Zeroize for G1Affine {
+    fn zeroize(&mut self) {
+        self.x.zeroize();
+        self.y.zeroize();
+        self.infinity = Choice::from(0);
+    }
+}
+
```

Comment:
Assuming that `Fp::zeroize` results in the value being zero, this manual impl places `G1Affine` into an invalid internal state: `(0, 0)` is not a valid BLS12-381 curve point. I think it would make more sense for the zeroization result to be the identity. Remove this:
```suggestion
```

and instead, right after the `impl Default for G1Affine` (which returns the identity), add:
```rust
#[cfg(feature = "zeroize")]
impl zeroize::DefaultIsZeroes for G1Affine {}
```